sdks/python/apache_beam/typehints/decorators.py

```diff
@@ -378,6 +378,43 @@ def has_simple_output_type(self):
         self.output_types and len(self.output_types[0]) == 1 and
         not self.output_types[1])
 
+  def strip_pcoll_input(self):
+    # type: () -> IOTypeHints
+
+    input_type = self.input_types[0][0]
```

Comment:
Please handle cases where self.input_types is None or the number of arguments is not 1.